Makefile

```diff
@@ -26,17 +26,11 @@ $(SQLITE_DB): var
 
 .PHONY: test
 test:
-	python manage.py test
+	tox
 
 .PHONY: quality
 quality:
-	pep8
-	pylint workbench/ sample_xblocks/ setup.py
-
-.PHONY: cover
-cover:
-	coverage run manage.py test
-	coverage report
```

Comment:
Do we want to keep a target for generating the coverage report?  The cookiecutter has one that launches the HTML report in a browser window.

Reply:
I had a version of it in the next round of changes, which pulls in most of the cookiecutter Makefile. Once this goes in I'll make a new branch of those changes and push it so whoever gets to this next has a starting place a little further along.